oci/handler/containerd.go

```diff
@@ -103,21 +105,21 @@ func (handle *ContainerdHandler) DigestExists(ctx context.Context, dgst digest.D
 }
 
 // ListManifests implements DigestResolver.
-func (handle *ContainerdHandler) ListManifests(ctx context.Context) ([]ocispec.Manifest, error) {
+func (handle *ContainerdHandler) ListManifests(ctx context.Context) (manifests []ocispec.Manifest, err error) {
 	ctx, done, err := handle.lease(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	defer done(ctx)
+	defer func() {
+		err = closeLease(ctx, done, err)
+	}()
```

Comment:
Does this alter the behaviour?

Reply:
It shouldn't.
The helper executes the `done` function like it used to, but now also does some error mangling to make sure that all errors are handled (and combined if necessary).

Reply:
ah, okay, makes sense, it got me confused about leases. We can leave it like this then

Reply:
ah you already changed, nice :D